tests/unit/evaluation/CrossValidatedCalibration_unittest.cc

```diff
@@ -0,0 +1,65 @@
+#include "environments/MultiLabelTestEnvironment.h"
+#include <gtest/gtest.h>
+#include <shogun/distance/EuclideanDistance.h>
+#include <shogun/evaluation/CrossValidatedCalibration.h>
+#include <shogun/evaluation/MulticlassAccuracy.h>
+#include <shogun/evaluation/MulticlassAccuracy.h>
+#include <shogun/evaluation/SigmoidCalibrationMethod.h>
+#include <shogun/evaluation/StratifiedCrossValidationSplitting.h>
+#include <shogun/features/DataGenerator.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/labels/MulticlassLabels.h>
+#include <shogun/multiclass/MulticlassOCAS.h>
+
+using namespace shogun;
+
+extern MultiLabelTestEnvironment* multilabel_test_env;
+
+#ifdef HAVE_LAPACK
```

Comment:
what is LAPACK needed for here?

Reply:
I borrowed this from another test. I will write proper test cases after we discuss the api. Or should I write them now?

Reply:
After API is frozen